streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamTransformValuesTest.java

```diff
@@ -138,6 +140,24 @@ public void close() { }
         assertArrayEquals(expected, supplier.theCapturedProcessor().processed.toArray());
     }
 
+    @Test
+    public void shouldEmitNoRecordIfTransformReturnsNull() {
+        final ProcessorContext context = mock(ProcessorContext.class);
+        final ValueTransformerWithKey<Integer, Integer, Integer> valueTransformer = mock(ValueTransformerWithKey.class);
+        final KStreamTransformValues.KStreamTransformValuesProcessor<Integer, Integer, Integer> processor = new KStreamTransformValues.KStreamTransformValuesProcessor<>(valueTransformer);
+        processor.init(context);
+        EasyMock.reset(context, valueTransformer);
```

Comment:
req: Could you please remove `context` from the `reset()`? Since we did not specify any behavior for it, we also do not need to reset it.

Reply:
I think that the `valueTransformer` is not strictly needed to be reset either, since it's not being verified, but there is a call to `ValueTransformer#init`, so I left it in the reset anyway.

Reply:
Good catch! I would prefer to remove it then.